integration-tests/src/test/java/oracle/verrazzano/weblogic/kubernetes/ItVzMiiDomain.java

```diff
@@ -197,7 +217,202 @@ void testCreateVzMiiDomain() {
     message = "Hello World, you have reached server managed-server";
     String appUrl = "https://" + host + "/sample-war/index.jsp --resolve " + host + ":443:" + address;
     assertTrue(verifyVzApplicationAccess(appUrl, message), "Failed to get access to sample application");
+
+  }
+
+  /**
+   * Create a WebLogic domain VerrazzanoWebLogicWorkload component in verrazzano with auxilary images.
+   */
+  @Test
+  @DisplayName("Verrazzano mill domain with auxilary image")
+  void testVzMiiAuxilaryImage() {
+    String auxDomainUid = "aux-domain";
+    String miiAuxiliaryImage1Tag = "image1" + MII_BASIC_IMAGE_TAG;
+    String miiAuxiliaryImage1 = MII_AUXILIARY_IMAGE_NAME + ":" + miiAuxiliaryImage1Tag;
+    String miiAuxiliaryImage2Tag = "image2" + MII_BASIC_IMAGE_TAG;
+    String miiAuxiliaryImage2 = MII_AUXILIARY_IMAGE_NAME + ":" + miiAuxiliaryImage2Tag;
     
+    final int replicaCount = 2;
+    final String adminServerPodName = auxDomainUid + "-admin-server";
```

Comment:
You can use ADMIN_SERVER_NAME_BASE here.

Reply:
fixed